app/src/main/assets/json.form/anc_addo_danger_signs.json

```diff
@@ -3,7 +3,7 @@
   "show_errors_on_submit": true,
   "skip_blank_steps": true,
   "count": "3",
-  "encounter_type": "ADDO Visit - Danger signs",
+  "encounter_type": "ANC Danger Signs",
```

Comment:
I have renamed this to Child ADDO Visit - Danger signs and am using the value in a method